packages/iTwinUI-react/src/core/Table/Table.test.tsx

```diff
@@ -576,9 +578,6 @@ it('should filter table', () => {
 
   setFilter(container, '2');
 
-  const tippy = document.querySelector('[data-tippy-root]') as HTMLElement;
-  expect(tippy.style.visibility).toEqual('hidden');
-
   rows = container.querySelectorAll('.iui-table-body .iui-row');
```

Comment:
no need to check same `.not.toBeVisible()` ?

Reply:
I'm checking in `setFilter`